qa/workunits/fs/full/subvol_rm_retained_snap_when_cluster_is_full.sh

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env bash
+set -ex
+
+# This testcase tests the scenario of the 'ceph fs subvolume rm --retained-snapshots'
+# mgr command when the osd is full.The osd is of the size 2GiB. The subvolume is
+# created and 1GB file is written. The full-ratios are set below 500MB such that the
+# osd is treated as full. Now the subvolume with retained-snapshots is removed. This
+# should fail with the introduction of FULL capabilities which the mgr holds as the
+# config update gets ENOSPACE error
```

Comment:
Reworded this test description in a better way and split it into 2: title of description and detailed description.

```suggestion
# Test that command 'ceph fs subvolume rm --retained-snapshots' fails when the
# OSD is full.
#
# A subvolume is created on a cluser with OSD size 2GB and a 1GB file is written on
# the subvolume. The OSD size is then set to below 500MB so that OSD is treated as
# full. Now the subvolume is removed but snapshots are retained. This should fail
# with the introduction of FULL capabilities which the mgr holds as the
# config update gets ENOSPACE error
```